notebooks/image_examples/image_classification/Front Page DeepExplainer MNIST Example.ipynb

```diff
@@ -290,58 +173,31 @@
LKbRqxKizqKCzXcvrG4TbXxLwb6fQfm3XT7ds96VikH/7azZm54P2X+WeQTv2tcD3a6beZ8P+Ps4wMI4w3XHiKZS/yvRxul2U6/KZZqdv9m+eO13LhMj/d5wTttFyporVPHB5uf//W+DrtbZSCdllNcZUadasOyMfcSH0zXUta13xkuh6NgfGZSNsygt94FkUGHQAAAAAAAAAAAAAAAKCLeEAHAAAAAAAAAAAAAAAA6CKWuAJwwfCXiEu4rcxq/jJxC6YTVXSZOJaIw4XkiScej+xbTtz8wjveIUnKeCaL4uvf8AZJ0mCOZ4Ox+u3atUuSNJ8aCfY1GiZbab2+cKreG2+8UZLkpEzK2yuu2Gre8A514zKBc8qOZ56RJFUPBJmtg3ipLbLc6O3vfa8kaT7RL0m66aabJEnFMpl0sfr5/Zsr98wG+2rNuHHcheNmof7NctKyA+c7P25y5dbqCo5j4qVWWzhufvGd75Qk5T0Tb4wL4ELy1FNPSZIGy8EqCa24cTovUSUtHDcLLWsFrDY7nnlG+bK9mk+tXpPjLR4D27Ztk6vW6iFveuObWBoHFwy/zmlXcxx5i9Q3khkbSKgVb7fccgv9G1wwnnrqKcUPT1n7XNdVzVt8ybh3vevdms22lrh60xvfpPFSZBU6dAG/kgEAAAAAAAAAAAAAAABdRAadc9i73vWuyL77779/yePe//73W9t33XXXil0T8HIqlj3rIcJyvRGZKlCuNawylw+nqkud1020DvEzgMRzA9Z5vPLUiky77jS7Ybzkhh6OjD4rGf5bS6G/U5IGc3GmhmNFHJmuR9oDG7KyHq/uFDfJ0ctT7WXSm27o8V9PptdLkoaco3OSNDFvnrzmvkVXVTvMeh6fSVjbzx/qi5TZd3yNtT1dGomUmU9Y9cuOv/6D/x0u8uCuyjpJOhS/WJJ0bf2poOG2+3vfkCR95MW9+fZjBmP5jP96oGTCY2S2bj7/O0+YNwrZSuR6iqWStf2/Hh6MlHnmgL3v0nWzkTIXj5at7UvWRuvRNf3ELaIyqfCe6H0SLvOjXblImb1jWUm65JC5pfdtzgdTfv7VPSFJ2hWvWMGdHLo4OLGfOaec6E1L0tisuYz+bCwyTSiXjFnT5wqp2Ond29G/HYgI92UkqVK32/QzVS9yMzlu3Poe7th2atZ3fmbQ8aF08F2+05uRJE0mGvZxHSbBzcd7k5J0ZNqEy2TZS4TLXLsuHa4Xlo514DR1+l4O94XnnWhshfvPlw+nFk6F4x+TSwSfNanmjOzJA6G4aZ3GzwDiZzTwZ2ST0QBnW6fsAJ2+z8OmKw1rHOA1G9OlcBn//s7KVAXlXOu0ftzEnXnrmPbxND9uZhNDKUnaPWFi6sRcp/rPjvUTc14mXKbm2bF+xWgqcs3hv4MYRSed4qbT2GtYpF32wpFoI2jrhqDymO5L6Ui/3S1/pjevuKNTHofudM2R+3tN/+KpEoAVFm67lZxGpP5JJewyb7ksOxMuE46/lGrWMZ4T+bpXPF2wjnFjKVVj+WDfkem6RguJSEwsbxzA/jro9P0Q/o0nn4722xgHv/CE75WTpWibp1i2963tSUTGYg9Oudb48b+/Mj0R+bBH9hf8l+tOVLR3c6HY/vYD7nE9581Y5w6viDCfjasazwdtwoWy50Tqm3wmWikN9XK/nwIy6AAAAAAAAAAAAAAAAABdRAYdAKtWOAvVcjJQbdu2TZKU8loJBchChQvJb/zGb0iS8hXzwPUDDzwgKfp0dbvbbrtNkjST2xjse8973iNJetXaJSfuAee93/u935MkFYdvlCTt37dfklQ/vmvBY2680ZQttC2h/aY3vVGSFN9/ogtXCZxbPv6JT0iS1pfNnJFnn31WkvRYo7jgMXfeeackqZwaCvb99m//tiTpeIm15bH6+f2bZLNZ5rfTLlF+oUOCuIk3WjPh/P7N0VkmuGH1O51xgcXixnGpb7D63f7e2yW1YuBMx9P2TnaejQ2sNh/5yEd0uNCqJ+6//37F+9cvesy2bdusuJFM7FDf4EJx+3tvl5fIBtsPPfSQGh0y5rS788475cbS1r4//M9/qPklcygCq8Pt732vrppq5WB5/PHtKjYWD4A777hD9YSdCeeuu+5iXOAsIYMOAAAAAAAAAAAAAAAA0EVk0DmHHDt2zNp+5JFHImWqVXspupGRkUiZ//Jf/ou13dPTswJXB3RXp3V0w3LJWKTQcD6x5HPRXqyZwSNhlnaMta1PGo+bmPLX+g3PWDhbwn/r8tZGBU5Pud5ax91LmNkH89nBpCTV+9eZN6aPRtbszV7xc8OSdLLnCklS0q0E8VjjjsW5quJE0zo5dXs94EaHtefz2dn2zcPTbiY4PDMsSfrQD+8tS9ILL7zgvxVM56nXTQhdFu8bkKQ1zWwf/ZVGn18m/tKYJCk2Xeo3O5qXOuhOR65nfHousi/M9ey/o1aPRcpUa3aZ0b7o9NZMZLlkIGo598muw0Gaj80lc+v960XuhCQ9sOOIJKnevy7oozrNvk9CvUlJchKmX1NN9gWxtVDmnI39ySXbiI8edgrhfVevTVmNQdaMx7mo3vzm9jMcvqC54H6vNppjBs1v8/YMIGGVmmkDxhfpixXLdl0ymEsREzir2vsuvk5jBSuhkuwP6iR/ZunYnJeSpFzOpEEczMUj/aTw9WzoT0bKACulVOvQd1mGTNJOk5tOtkJrwboj0dbeS5jDN1ScuCStnTPZDibz0bbTVLWRlKRq8y3HjV7zSCFutd029ieq4TK7xutWyriDU/VsuEz477ppY+bcGOjDeWdZY7Gh/vN0uiEnPRjUHQtlz/HHoaXlj0V3ivV0MnSN9N3xMmuvOyRp80CystQx4XvZS2RVT/UEY3XxwrDcthiRJLk1JUcvD27wcmrIGheQpJ3jnlUH5Fy3Y7uxWPas38U7teWG8wkrtjt9HwzmkvSLLnCdftOcd+z+85zTiIxDT8x7mfbt+Vq0TCoe6t+UKt
```

Comment:
shouldn't this be 5 predictions, since you changed the slicing from `[1:5]` to `[0:5]`

Reply:
Yes, you are right It should be 5. I will update this in the new commit.